apps/discord-bot/internal/command/command.go

```diff
@@ -22,10 +23,13 @@ type Command interface {
 }
 
 type commands struct {
+	twitchAccessToken string
 }
 
-func NewCommands() Command {
-	return &commands{}
+func NewCommands(token string) Command {
+	return &commands{
+		twitchAccessToken: token,
+	}
 }
 
 func (c *commands) GetCommands() map[string]func(context context.Context, s *discordgo.Session, i *discordgo.InteractionCreate, db db.MySQL) {
```

Comment:
We should create a return type for GetCommands() like we did in twitch-bot

Reply:
I'll fix it in the next pr